src/pymmcore_widgets/_mda/_time_plan_widget.py

```diff
@@ -180,6 +181,7 @@ def _create_new_row(
         time_spin.setValue(loops or 1)
         time_spin.setButtonSymbols(QAbstractSpinBox.ButtonSymbols.NoButtons)
         time_spin.setAlignment(Qt.AlignmentFlag.AlignCenter)
+        time_spin.setKeyboardTracking(False)
```

Comment:
could probably also stand to bring a similar thing to all the other subcomponents like here:
https://github.com/pymmcore-plus/pymmcore-widgets/blob/58f35f4c3fdcdf8e830c301d4a3de8215c1a948b/src/pymmcore_widgets/_mda/_channel_table_widget.py#L263-L264